matrix-sdk/src/main/java/org/matrix/androidsdk/rest/model/RoomMember.java

```diff
@@ -166,6 +168,20 @@ public void setInviterId(String userId) {
         mInviter = userId;
     }
 
+    public String getAvatarUrl() {
+        // allow only url which starts with mxc://
+        if ((null != avatarUrl) && !avatarUrl.startsWith(ContentManager.MATRIX_CONTENT_URI_SCHEME)) {
```

Comment:
This test will fail if the avatar url starts with MXC: (or Mxc:, etc.) for some odd reason. I don't know how bad that is, but it's something to consider.